core/src/main/java/org/elasticsearch/action/termvectors/TermVectorsResponse.java

```diff
@@ -305,9 +305,9 @@ private void buildFieldStatistics(XContentBuilder builder, Terms curTerms) throw
         long sumDocFreq = curTerms.getSumDocFreq();
         int docCount = curTerms.getDocCount();
         long sumTotalTermFrequencies = curTerms.getSumTotalTermFreq();
-        if (docCount > 0) {
-            assert ((sumDocFreq > 0)) : "docCount >= 0 but sumDocFreq ain't!";
```

Comment:
:) it's a good sign you bumped into this, isn't this changing semantics -  before when there were no docs, we won't output the Field Stats sub object

Reply:
expand the diff below and you will see all the void 💃